index.d.ts

```diff
@@ -25,6 +25,14 @@ declare namespace NodeVault {
             res?: Option;
         };
     }
+    interface VaultError extends Error { }
+
+    interface ApiResponseError extends VaultError {
+        response: {
+            statusCode: number,
+            body: any
```

Comment:
Handl vault response is using a structure that is similar to the one in ApiResponseError, Do we need a type for this as well?